src/Components/animation/ProgressBar.js

```diff
@@ -174,9 +174,17 @@ class ProgressBar extends React.Component {
   }
 
   _stepButtonClicked = key => {
-    Animated.parallel(
-      this.props.getStepAnimation(this.progressBarDots[key].key, this.props.currentStepAV <= key),
-    ).start();
+    var isNextStep = key - this.props.currentStepAV._value <= 1 && key - this.props.currentStepAV._value > 0;
```

Comment:
C'est équivalent à faire `key === this.props.currentStepAV._value + 1` ?

Reply:
Pas tout à fait.
Comme currentStepAV est une AnimatedValue, il prend des valeurs non entières au fur et à mesure de l'animation (ça permet de faire avancer la barre de progression par exemple)

Reply:
> C'est juste un peu bizarre quand on change d'étape dans l'éditeur, il faut attendre un peu

Oui ça doit être le temps que l'animation s'effectue entre ces deux étapes. Tu penses que c'est mieux si c'est toujours instantané ? (ça doit être souvent mieux effectivement...)

Par contre si on est à une étape "stepId" et qu'on va à l'étape "stepId + 1" c'est peut être bien si l'animation n'est pas instantanée ?

Reply:
Je ne sais pas trop... Mais je dirais qu'à choisir, il vaut mieux dégrader un peu l'expérience dans l'éditeur si ça permet d'améliorer dans l'animation d'un drill